visualizer/actionButton.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-env browser */
+
+'use strict';
+
+(function() {
+
+  function selectNode(actionNode) {
+    actionNode.classList.add('selected');
+    if (!actionNode.readOnly) {
+      actionNode.select();
+    }
+  }
+
+  function addAction(actionType) {
```

Comment:
This code is very nice and readable! Good job with formatting and naming.

In the future, you might want to add at least a few short comments describing the most important functions, and/or a comment describing the purpose of the file. Even when the code is nice and simple like this, the comments give a good starting point for reading the code, and makes it easier to have an overview of the code at a glance.